web/pages/program/[slug].tsx

```diff
@@ -66,6 +69,7 @@ interface SessionRouteProps {
     home: {
       ticketsUrl: string;
     };
+    navItems: PrimaryNavItem[];
```

Comment:
What happens if `navItems` is `null` or `undefined`?

Reply:
Fixed in 619a555. :)